ZelBack/src/lib/server.js

```diff
@@ -4,8 +4,15 @@ const cors = require('cors');
 const morgan = require('morgan');
 const compression = require('compression');
 
-const expressWs = eWS(express());
-const { app } = expressWs;
+// see https://github.com/HenningM/express-ws/issues/120
+const options = {
+  wsOptions: {
+    maxPayload: 65535,
```

Comment:
is this sufficient?

Reply:
It may not be for the terminal. Worked fine during testing but I wasn't sending big payloads. Maybe we should set this to something like 1_048_576 * 16 (16Mb)

Reply:
Agreed lets try to set it to that higher value. Can you make a PR for it?

Reply:
Yup - will do this now

Reply:
Done